tests/test_stack.py

```diff
@@ -41,3 +41,12 @@ def test_items_removed_in_lifo_order(stack):
     assert stack.pop() == 3
     assert stack.pop() == 5
     assert stack.empty()
+
+def test_str_with_items(stack):   
+    stack.push(5)
+    stack.push(3)
+    stack.push(7)
+    assert stack.__str__() == [7, 3, 5]
+
+def test_str_with_empty_stack(stack):   
+    assert stack.__str__() == []
```

Comment:
A smilar change here
```suggestion
    assert str(stack) == "[]"
```